src/ui/Speed.tsx

```diff
@@ -30,10 +35,27 @@ const Foreground = (props: React.SVGProps<SVGSVGElement>): JSX.Element => (
   </svg>
 )
 
+const NitroBar = ({ boostRemaining, boostColor, ...props }: NitroProps): JSX.Element => (
+  <svg width={289} height={55} viewBox="0 0 289 55" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <path className="nitro-bg-path" d="M13,12 L200,12" />
+    <path
+      className={`nitro-path ${boostRemaining <= 65 ? 'nitro-path-blink' : ''}`}
+      stroke={boostRemaining > 80 ? '#00ff00' : boostColor}
+      strokeDasharray={`${boostRemaining * 3.9}px`}
```

Comment:
I am not a svg expert but how I googled few times strokeDasharray was used in path for the length manipulation. But if you have a better approach please post that here.

Reply:
Il do it in a later refactor, I've seen some other svg's in the code using some properties that shouldn't be used that way.

Reply:
I'll be happy to see later a better way. I love svg but I am really a rooky on that sector :P

Reply:
> I love svg but I am really a rooky on that sector :P

SVG's are awesome but can be very overwhelming as in theory you can use `path` for everything and certain properties like this can behave _correctly_ and manipulate the length. Shouldn't be much of a problem anyway, but paths can be heavy as browsers need to calculate positions of `d=` etc.

Reply:
yes true :)